manifests/scripts/metrics-dashboard/README.md

```diff
@@ -0,0 +1,7 @@
+# Enable Prometheus logging of GitOps Service in 'gitops' namespace, and install Grafana 
+
+### How to Use
+1) Acquire an OpenShift Cluster (for example, cluster bot) and log into it.
+2) Install OpenShift GitOps operator to it.
```

Comment:
Should that be the GitOps service operator?  If not, when does that operator get installed?

Reply:
I've updated the README.md with more information about what is being exposed.